modules/javafx.graphics/src/main/java/com/sun/glass/utils/NativeLibLoader.java

```diff
@@ -29,14 +29,21 @@
 import java.security.AccessController;
 import java.security.PrivilegedAction;
 import java.util.HashSet;
+import java.io.*;
+import java.nio.*;
+import java.nio.file.*;
 
 public class NativeLibLoader {
 
     private static final HashSet<String> loaded = new HashSet<String>();
+    private static final String tmpdir = System.getProperty("java.io.tmpdir");
 
     public static synchronized void loadLibrary(String libname) {
         if (!loaded.contains(libname)) {
-            loadLibraryInternal(libname);
+            StackWalker walker = AccessController.doPrivileged((PrivilegedAction<StackWalker>) () ->
+            StackWalker.getInstance(StackWalker.Option.RETAIN_CLASS_REFERENCE));
```

Comment:
For this method youdon't need `AccessController.doPrivileged` since the caller needs to assert privileged before calling. It isn't harmful, though.